### PR TITLE
namespace updated to kubescape(default) for kubescape-host-scanner ds

### LINKED ISSUE #61: namespace consolidation is required for kubescape-cloud-operator
# Description
kubescape-host-scanner is deployed into a different namespace i.e. ```kubescape-host-scanner```. Kubescape should deploy all Kubernetes objects into a single namespace. Also, only for DaemonSet of ```kubescape-host-scanner``` namespace is defined (hard-coded) into its manifest, while for other Kubernetes objects, the namespace is inherited from ```values.yaml```
 
# Environment
Kubernetes Version: ```v1.23.13-eks-fb459a0```
Kubescape Version: ```kubescape-cloud-operator(1.9.3)```
 
# Steps To Reproduce

Deploy the ```kubescape-cloud-operator``` Helm Chart. kubescape-host-scanner will be deployed into a different namespace i.e. ```kubescape-host-scanner```.
# Expected behavior

Kubescape Cloud Operator should deploys all Kubernetes objects into a single namespace ```kubescape``` by default.

# Actual Behavior

kubescape-host-scanner is deployed into a different namespace i.e. ```kubescape-host-scanner```.

# Additional context

### PR DESCRIPTION
**Pull request by nikhilmaheshwari24**
## Overview

Removed the manifest to create namespace from file ```charts/kubescape-cloud-operator/assets/host-scanner-definition.yaml``` and updated the manifest of ```kubescape-host-scanner``` DaemonSet for namespace.

## How to Test

Deploy the ```kubescape-cloud-operator```.
 
## Related issues/PRs:

* Resolved #61 

## Checklist before requesting a review

- [x] My code follows the style guidelines of this project
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: charts/kubescape-cloud-operator/assets/host-scanner-definition.yaml [hunk 1]
```diff
@@ -1,18 +1,9 @@
...
-    kubernetes.io/metadata.name: kubescape-host-scanner
-    tier: kubescape-host-scanner-control-plane
-  name: kubescape-host-scanner
 ---
 apiVersion: apps/v1
 kind: DaemonSet
 metadata:
   name: host-scanner
-  namespace: kubescape-host-scanner
+  namespace: {{ .Values.ksNamespace }}
```
Comment (Bezbran) [inline charts/kubescape-cloud-operator/assets/host-scanner-definition.yaml:6]: We need forst to upadte kubescape code so it won't delete the host-scanner namespace once it finished.

Comment (Bezbran) [reply charts/kubescape-cloud-operator/assets/host-scanner-definition.yaml:6]: https://github.com/kubescape/kubescape/blob/2a0a2cf95ab59a54ab190179bb33f837a02447bc/core/pkg/hostsensorutils/hostsensordeploy.go#L279

Comment (Bezbran) [reply charts/kubescape-cloud-operator/assets/host-scanner-definition.yaml:6]: And we need to fix the YAML in kubescape itself as well:
https://github.com/kubescape/kubescape/blob/master/core/pkg/hostsensorutils/hostsensor.yaml

**Review by Bezbran — CHANGES_REQUESTED** [review]

**Comment by dwertent** [comment]
Merged in #149

**Pull request closed without merging**